src/api/index.js

```diff
@@ -56,22 +60,22 @@ const getDSData = (url, authState) => {
     .catch(err => err);
 };
 
-const apiAuthGet = (endpoint, authHeader) => {
-  return axios.get(`${getApiUrl()}${endpoint}`, { headers: authHeader });
+const apiAuthGet = endpoint => {
+  return axios.get(`${getApiUrl()}${endpoint}`, getAuthHeader());
```

Comment:
I like the changes here where a middleware function is being passed to take care of the state being passed. My question is, on line 53 it looks like getAuthHeader() was assigned to headers. Is that necessary if we are passing in the actual function here? Can we just pass the variable 'headers'?